ewfs/noise_models.py

```diff
@@ -1,22 +1,32 @@
 """Custom noise models for simulator-based experiments."""
+from typing import Set
+
+from qiskit import QuantumCircuit
+
 from qiskit_aer.noise import (
     depolarizing_error,
     NoiseModel,
     pauli_error,
 )
 
 
-def depolarizing_noise_model(error: float = 0.01) -> NoiseModel:
+def depolarizing_noise_model(single_qubit_error_rate: float=0.01, two_qubit_error_rate: float=0.03, 
+                             single_qubit_gates: Set=None, two_qubit_gates: Set=None) -> NoiseModel:
     """Defines an depolarizing noise model with one-qubit.
 
     Args:
         error: One-qubit gate error rate (default 1%).
     Returns:
         Depolarizing noise model.
     """
+    if single_qubit_gates == None:
+        single_qubit_gates = ["u1", "u2", "u3"]
+    if two_qubit_gates == None:
+        two_qubit_gates = ["cx"]
```

Comment:
Slightly more concise and uses `is` instead of `==` to type-check `None`. 

```suggestion
    single_qubits_gates = ["u1", "u2", "u3"] if None else single_qubit_gates
    two_qubit_gates = ["cx"] if None else two_qubit_gates
```